Docs/articles/Kernel/CGS.md

```diff
@@ -41,19 +43,19 @@ namespace GraphicTest
         protected override void BeforeRun()
         {
             /* If all works correctly you should not really see this :-) */
-            Console.WriteLine("Cosmos booted successfully. Let's go in Graphic Mode");
-
-            canvas = FullScreenCanvas.GetFullScreenCanvas();
+            Console.WriteLine("Cosmos booted successfully. Let's go in Graphical Mode");
 
+            canvas = FullScreenCanvas.GetFullScreenCanvas(new Mode(640, 480, ColorDepth.ColorDepth32));
```

Comment:
Please add a comment that it is not necessary to specify the mode if the standard more should be used.